samples/unit-tests/legend/legend-general/demo.js

```diff
@@ -130,6 +130,19 @@ QUnit.test('Legend redraws', function (assert) {
     chart.series[0].hide(true);
 
     assert.strictEqual(visible, false, 'Legend item text has changed (#2165)');
+
+    chart.series[0].remove(false);
+
+    chart.update({
+        legend: {
+            title: {
+                text: 'Legend'
+            }
+        }
+    });
+
+    assert.strictEqual(chart.legend.group.attr('visibility'), 'hidden',
+        'Legend title is not visible (#17167)');
```

Comment:
Instead of this:
```js
someFunction(first, second,
    third, fourth);
```

It's better to read and looks much better this way:
```js
someFunction(
    first,
    second,
    third,
    fourth
);
```

(see similar assertions above)